internal/gps/pkgtree/pkgtree.go

```diff
@@ -78,10 +87,13 @@ func ListPackages(fileRoot, importRoot string) (PackageTree, error) {
 		case "vendor", "Godeps":
 			return filepath.SkipDir
 		}
-		// We do skip dot-dirs, though, because it's such a ubiquitous standard
-		// that they not be visited by normal commands, and because things get
-		// really weird if we don't.
-		if strings.HasPrefix(fi.Name(), ".") {
+
+		// Skip dirs that are known to be VCS roots.
+		//
+		// Note that there are some pathological edge cases this doesn't cover,
+		// such as a user using Git for version control, but having a package
+		// named "svn" in a directory named ".svn".
```

Comment:
Not sure if we want to try and address this -- should we only skip these dirs if we're at the root of the project? Will that cause problems for submodules?